Tests/FoundationEssentialsTests/PredicateTests.swift

```diff
@@ -14,6 +14,13 @@
 import TestSupport
 #endif
 
+#if !FOUNDATION_FRAMEWORK
+// Resolve ambiguity between Foundation.#Predicate and FoundationEssentials.#Predicate
+@freestanding(expression)
+@available(macOS 14, iOS 17, tvOS 17, watchOS 10, *)
+macro Predicate<each Input>(_ body: (repeat each Input) -> Bool) -> Predicate<repeat each Input> = #externalMacro(module: "FoundationMacros", type: "PredicateMacro")
+#endif
+
```

Comment:
Sigh... we can't switch to swift-testing fast enough.. do we have to do this for every test file?

Reply:
Luckily this is the only file that uses this macro so at least for now it's a non-issue, and I _think_ it still might work in other files too (since the `macro`'s visibility is `internal`) but I'm not certain